sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/StoreType.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// Time Series Insights store type.
```

Comment:
store meaning storage of data?

Reply:
Yup. Maybe change to something like:
"Time Series Insights data storage type."?